data_test/HP_MockData.csv

```diff
@@ -0,0 +1,18 @@
+Name,Overall GPA,CS GPA,"Overall Programming Experience (1 - Basic, 2 - Good, 3 - Expert)",Video Game Development Experience,Database Development Experience,Embedded Systems Software Development,Web Application Development Experience,Mobile Application Development Experience,UI or EX Experience,Statistics Experience,Networking Experience,Security Experience,Skill Improvement,First Preference,Second Preference,Third Preference,"IP Preference (1 - keep, 2 - depends, 3 - don't care)","Leadership Role (1 - only leader, 2 - prefer to lead but will follow, 3 - either works, 4 - prefer to follow but can lead, 5 - will only follow)",Prefer to work with,Prefer not to work with
```

Comment:
I like the idea of adding this into the master branch (maybe not totally production legit, but I think it's good for the whole team to have access)